WasabiWords.lua

```diff
@@ -298,7 +322,12 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
         stats.set_int('MPX_CHAR_HAIR_UNLCK9', -1)
         stats.set_int('MPX_CHAR_HAIR_UNLCK10', -1)
         stats.set_int('MPX_CHAR_HAIR_UNLCK11', -1)
-        stats.set_int('MPX_CHAR_HAIR_UNLCK1', -1)
```

Comment:
Why was this removed?

Reply:
Just because it was a duplicate. It was probably meant to be `MPX_CHAR_HAIR_UNLCK12`, but the 2 was missing. The same stat is still at the beginning of that block.

Reply:
Yeah I saw when I checked the file. Thanks for the contribution.